### PR TITLE
add de.zwarf.picplanner

### PR DESCRIPTION
**Pull request by zwarf12**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: de.zwarf.picplanner.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            "config-opts" : [
+                "-Dglade_catalog=false",
+                "-Denable_vala=false",
+                "-Dgtk_doc=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libgweather.git",
+                    "branch" : "main",
```
Comment (hfiguiere) [inline de.zwarf.picplanner.json:54]:
```suggestion
```

File: de.zwarf.picplanner.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                "-Ddemos=true",
+                "-Dgtk_doc=false",
+                "-Dvector_renderer=true"
+            ],
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libshumate.git",
+                    "branch" : "main",
```
Comment (hfiguiere) [inline de.zwarf.picplanner.json:101]:
```suggestion
```

File: de.zwarf.picplanner.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        },
+        {
+            "name" : "picplanner",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/Zwarf/picplanner.git",
+                    "branch" : "main",
```
Comment (hfiguiere) [inline de.zwarf.picplanner.json:114]:
```suggestion
```

File: de.zwarf.picplanner.json [hunk 4]
```diff
@@ -0,0 +1,123 @@
+{
+    "app-id" : "de.zwarf.picplanner",
```
Comment (hfiguiere) [inline de.zwarf.picplanner.json:2]: Does that app-id mean the requirements? I get "Diese Domain wurde geparkt." on `zwarf.de`

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build de.zwarf.picplanner

File: de.zwarf.picplanner.json (hunk 4 above)
Comment (zwarf12) [reply de.zwarf.picplanner.json:2]: I own this domain. The website for PicPlanner will be online soon.

**Comment by hfiguiere** [comment]
bot, build de.zwarf.picplanner

File: de.zwarf.picplanner.json (hunk 4 above)
Comment (zwarf12) [reply de.zwarf.picplanner.json:2]: I placed a "place holder" website until the normal website is done.

**Comment by nedrichards** [comment]
bot, build de.zwarf.picplanner

**Comment by nedrichards** [comment]
bot, build de.zwarf.picplanner

**Pull request closed without merging**